components/playlist/browser/resources/playlist.html

```diff
@@ -14,6 +14,7 @@
 
   body {
     margin: 0;
+    background-color: var(--leo-color-container-background);
```

Comment:
This is 100% fine as is, but thought I'd let you know - there's a [createGlobalStyle](https://styled-components.com/docs/api#createglobalstyle) utility in styled components

Reply:
Thanks @fallaciousreasoning ! I tried that but somehow it didn't work but not sure where I made a mistake. Could you take a look at https://github.com/brave/brave-core/compare/sko/pl-background-test?expand=1 ?

<img width="379" alt="image" src="https://github.com/brave/brave-core/assets/5474642/b908cf15-7a13-4be1-b535-1a92b2567a5b">

Reply:
Oh weird, I would've thought that would work :laughing: - I'll have a play and see if I can work it out, but this is fine as is :smile:

Reply:
(I'm marking this as resolved, while I faff around with it)

Reply:
could this be related to CSP of chrome-untrusted:// frame?